benchmarks/huggingface/bench/__main__.py

```diff
@@ -108,6 +113,14 @@ def parser():
         dest="allow_tf32",
         action="store_false",
         help="do not allow tf32",
+        default=True,
+    )
+    parser.add_argument(
+        "--tf32",
+        dest="allow_tf32",
+        action="store_true",
+        default=True,
+        help="Allow tf32",
```

Comment:
I still find it a bit confusing to have both `--tf32` and `--no-tf32`, could we use something like `--precision tf32` instead?

Reply:
I added the --precision argument